focus-areas/communal-value/social-currency-metric-system.md

```diff
@@ -1,22 +1,23 @@
 # Social Currency Metric System (SCMS)
 
 Question: How does one measure the value of community interactions and accurately gauge “reputation” of a community as evident from qualitative sentiment?
+*Note: This metric synthesizes several candidate atomic metrics that can be derived from trace data, and several candidate process oriented atomic metrics. Embedded footnotes annotate areas planned for later clarification, and questions for later resolution.*
```

Comment:
I recommend we skip "atomic" as a concept until we centralize this terminology across working groups and differentiate them from "synthetic."

```suggestion
*Note: This metric synthesizes several other metrics that can be derived from trace data, and several process-oriented metrics. Embedded footnotes annotate areas planned for later clarification, and questions for later resolution.*
```